examples/minimal.rs

```diff
@@ -0,0 +1,166 @@
+use image::{ImageBuffer, RgbaImage};
```

Comment:
Should I do anything else to make the example program discoverable? Maybe a link from README.md or the lib.rs module docs?